crates/bevy_math/src/curves/variable.rs

```diff
@@ -0,0 +1,708 @@
+use std::cmp::Ordering;
+
+use crate::{
+    curves::{Curve, CurveCursor, CurveError},
+    interpolation::{Interpolate, Interpolation},
+};
+
+/// Keyframe tangents control mode
```

Comment:
This doc comment isn't very useful to users who don't know what tangent control does.

Reply:
done

Reply:
better comments